src/Importify/Common.hs

```diff
@@ -155,3 +157,23 @@ importSlice []               = Nothing
 importSlice [ImportDecl{..}] = Just $ startAndEndLines importAnn
 importSlice (x:y:xs)         = Just $ startAndEndLines
                                     $ combSpanInfo (importAnn x) (importAnn $ NE.last (y :| xs))
+
+-- | Returns module name of the source file
+-- We can't parse the whole file to get it because default extensions are not available yet
+getModuleName :: Text -> String
+getModuleName src = case impl of
+                        [mn] -> L.init mn
+                        []   -> error "File doesn't have `module' declaration"
```

Comment:
Well, better approach is too avoid parsing at all. When you're reading file and have full access to path of this file you can convert part of the path to module name. So in ideal case it would be better to pass or to get from somewhere path of this module and then use `simpleParse` from suggestions in this documentation: https://www.stackage.org/haddock/lts-8.16/Cabal-1.24.2.0/Distribution-ModuleName.html#v:fromString
I hope to avoid writing manual parser for module preamble...